tests/utils/test_cdb_utils.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+import numpy as np
+from tests.helper import ForCDBMerging
+from medcat.utils.cdb_utils import cdb_utils
+
+
+class CDBMergeTests(unittest.TestCase):
+    @classmethod
+    def setUp(cls) -> None:
```

Comment:
This is wrong. It's either a `@classmethod def setupClass(cls)` or a regular method `def setUp(self)`.